helm-charts/codegen-openshift-rhoai/tgi/templates/servingruntime-magicoder-template.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  annotations:
+    opendatahub.io/apiProtocol: REST
+    opendatahub.io/modelServingSupport: '["single"]'
+  labels:
+    opendatahub.io/dashboard: "true"
+  name: tgi-magicoder-s-ds-6.7b-cpu
+  namespace: redhat-ods-applications
+objects:
+- apiVersion: serving.kserve.io/v1alpha1
+  kind: ServingRuntime
+  labels:
+    opendatahub.io/dashboard: "true"
+  metadata:
+    annotations:
+      openshift.io/display-name: Text Generation Inference Magicoder-S-DS-6.7B on CPU
+    name: tgi-magicoder-s-ds-6.7b-cpu
+  spec:
+    containers:
+      - args:
+          - --model-id
+          - /mnt/models/--ise-uiuc--Magicoder-S-DS-6.7B/snapshots/b3ed7cb1578a3643ceaf2ebf996a3d8e85f75d8f
+          - --port=8080
+          - --json-output
+        env:
+          - name: NUMBA_CACHE_DIR
+            value: /tmp/hf_home
+          - name: HF_HOME
+            value: /tmp/hf_home
+          - name: HF_HUB_CACHE
+            value: /mnt/models
+          - name: HUGGING_FACE_HUB_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: HUGGING_FACE_HUB_TOKEN
+                name: hf-token
+          - name: BATCH_BUCKET_SIZE
+            value: "22"
+          - name: PREFILL_BATCH_BUCKET_SIZE
+            value: "1"
+          - name: MAX_BATCH_PREFILL_TOKENS
+            value: "5102"
+          - name: MAX_BATCH_TOTAL_TOKENS
+            value: "32256"
+          - name: MAX_INPUT_LENGTH
+            value: "1024"
+          - name: PAD_SEQUENCE_TO_MULTIPLE_OF
+            value: "1024"
+          - name: MAX_WAITING_TOKENS
+            value: "5"
+          - name: OMPI_MCA_btl_vader_single_copy_mechanism
+            value: none
+        image: ghcr.io/huggingface/text-generation-inference:2.1.0
```

Comment:
I wonder if we should make it as a variable from values.yaml file. It would be easier to change version in future